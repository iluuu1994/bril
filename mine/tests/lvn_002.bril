@main {
  a: int = const 1;
  b: int = const 2;
  c: int = add a b;
  print c;
  b: int = call magic_num;
  c: int = add a b;
  print c;
}
@magic_num {
  x: int = const 42;
  ret x;
}
