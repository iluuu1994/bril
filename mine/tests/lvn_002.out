@main {
  a: int = const 1;
  c: int = const 3;
  print c;
  b: int = call magic_num;
  c: int = add a b;
  print c;
}
@magic_num {
  x: int = const 42;
  ret x;
}
